DataFormats/Scouting/interface/Run3ScoutingTrack.h

```diff
@@ -0,0 +1,191 @@
+#ifndef DataFormats_Run3ScoutingTrack_h
+#define DataFormats_Run3ScoutingTrack_h
+
+#include <vector>
+
+//class for holding track information, for use in data scouting
+class Run3ScoutingTrack {
```

Comment:
Could you add the reminder on backwards compatibility the other classes have?
```suggestion
// IMPORTANT: the content of this class should be changed only in backwards compatible ways!
class Run3ScoutingTrack {
```